dask_expr/tests/test_merge.py

```diff
@@ -996,6 +996,33 @@ def test_merge_leftsemi():
         df1.merge(df2, how="leftsemi", on="aa")
 
 
+@pytest.mark.xfail(
+    _backend_name() != "cudf", reason="leftanti joins not supported with pandas backend"
+)
+def test_merge_leftanti_cudf():
+    pdf1 = pd.DataFrame({"aa": [1, 2, 3, 4, 5, 6, 1, 2, 3], "bb": 1})
+    pdf2 = pd.DataFrame({"aa": [1, 2, 2, 4, 4, 10], "cc": 1})
+
+    df1 = from_pandas(pdf1, npartitions=2)
+    df2 = from_pandas(pdf2, npartitions=2)
+    assert_eq(
+        df1.merge(df2, how="leftanti"),
+        pdf1[~pdf1.aa.isin(pdf2.aa)],
+        check_index=False,
+    )
+    df2 = df2.rename(columns={"aa": "dd"})
+    assert_eq(
+        df1.merge(df2, how="leftanti", left_on="aa", right_on="dd"),
+        pdf1[~pdf1.aa.isin(pdf2.aa)],
```

Comment:
Seems like we could just do this in [merge_chunk](https://github.com/dask/dask/blame/3432f4601eb7edc9cd5c2d64fd463d23ab4c3b25/dask/dataframe/multi.py#L244) for pandas data to support `how="leftanti"` for cpu as well.

Reply:
Yeah good point, can look into this a bit more

Reply:
Pushed some commits to https://github.com/dask/dask/pull/11150 that, in conjunction with this PR, should unblock left anti/semi joins on CPU